app/controllers/trainees/check_details_controller.rb

```diff
@@ -7,6 +7,7 @@ class CheckDetailsController < BaseController
     def show
       page_tracker.save_as_origin!
       @form = Submissions::TrnValidator.new(trainee: trainee)
+      @trainee_editable = trainee_editable?
```

Comment:
I believe we can make do with just calling the `trainee_editable?` helper, which should initialise  `@trainee_editable` automatically
```suggestion
      trainee_editable?
```

Reply:
Thanks Arlston - the reason we added it in this way with a variable was just to make it look a bit more 'normal' - but happy to change it back if you think that makes more sense!